apps/aecore/src/aec_conductor.erl

```diff
@@ -421,6 +422,14 @@ set_stratum_mode(State) ->
             State#state{stratum_mode = false}
     end.
 
+set_is_fork_configured(State) ->
+    IsForkConfigured =
```

Comment:
It may be more readable to call a helper function in `aec_fork_signalling` module here. Though debatable so ok.